src/commands/app/create.ts

```diff
@@ -69,6 +70,9 @@ export default class Create extends BaseCommand<typeof Create> {
       char: "d",
       description: commonMsg.CURRENT_WORKING_DIR,
     }),
+    org: flags.string({
+      description: appCreate.ORG_UID_DESCRIPTION,
+    }),
```

Comment:
@harshithad0703 , Since we're using base-command, there's no need to include the org flag here. 
In order to avoid duplicating common flags, please check with the doc team and request that they give a common message.

Reply:
removed common duplicating flag, will adding common message for org flag